docs/src/docs/deploy.md

```diff
@@ -0,0 +1,112 @@
+
+## Init Container
+
+The init container unpacks the model reference from a ModelKit to a specific path and then exits. This makes it useful as a Kubernetes `init` container. This container also supports verifying signatures for containers automatically from key-based or keyless signers.
+
+The container is configurable via environment variables:
+
+`$MODELKIT_REF`: The ModelKit to pull (required).
+`$UNPACK_PATH`: Where to unpack the ModelKit (normally you’d want a `volumeMount` here). This is required and will default to `/home/user/modelkit`.
+`$UNPACK_FILTER`: Optional filter to limit what is unpacked (e.g., just the model, or model + code). The filter format is the same as the [–filter command line argument](./cli/cli-reference.md) for the Kit CLI.
+`$COSIGN_KEY`: Path to the key that should be used for verification, mounted inside the init container (e.g., from a Kubernetes secret).
+`$COSIGN_CERT_IDENTITY`: Signing identity for keyless signing.
+`$COSIGN_CERT_OIDC_ISSUER`: OIDC issuer for keyless signer identity.
+
+### Example Kubernetes YAML
+
+```
+ apiVersion: v1
+ kind: Pod
+ metadata:
+   name: my-modelkit-pod
+ spec:
+   containers:
+     - name: model-server
+       image: "" # Some container that expects your modelkit
+       # Share a volumeMount between the init container and this one
+       volumeMounts:
+         - name: modelkit-storage
+           mountPath: /my-modelkit
+
+   # Run the init container to unpack the ModelKit into the volume mount and make
+   # it available to the main container
+   initContainers:
+     - name: kitops-init
+       image: ghcr.io/jozu-ai/kitops-init-container:latest
+       env:
+         - name: MODELKIT_REF
+           value: "ghcr.io/jozu-ai/my-modelkit:latest"
+         - name: UNPACK_PATH
+           value: /tmp/my-modelkit
+       volumeMounts:
+         - name: modelkit-storage
+           mountPath: /tmp/my-modelkit
+
+   # Define a volume to store the ModelKit
+   volumes:
+     - name: modelkit-storage
+       emptyDir: {}       
+```
+
+## Using the Kit CLI Container
+
+The containerized Kit CLI can be used to tailor the running of a ModelKit because you can run any Kit CLI command. This gives you flexibility, but more manual work (the world is your oyster, but it may be hard to shuck).
+
+This container runs `kit` as its entrypoint, accepting Kit CLI arguments. You can also replace any time you use a kit CLI command with `docker run ghcr.io/jozu-ai/kit:latest` and it will run Kit inside the container. This means if you want to download it to your local machine, you’ll need to mount a docker volume, but you can use this to inspect ModelKits without installing the CLI.
+
+Docker run example:
+
+`docker run ghcr.io/jozu-ai/kit:latest pull jozu.ml/jozu/llama3-8b:8B-instruct-q5_0`
+
+Kubernetes example:
+
+```
+ apiVersion: v1
+ kind: Pod
+ metadata:
+   name: my-modelkit-pod
+ spec:
+   containers:
+     - name: me-using-kit
+       image: ghcr.io/jozu-ai/kit:latest
```

Comment:
same here with the host (jozu.ml vs ghcr.io)

Reply:
The Kit CLI image is currently only available through ghcr.io and isn't published to jozu.ml (if I'm remembering right).